.gitignore

```diff
@@ -26,6 +26,8 @@ lang
 .mp_svn
 /nbproject/
 tests/_data/acceptanceGenerated.sql
+tests/acceptance/_groups/*
+!tests/acceptance/_groups/.gitkeep
```

Comment:
We could do this as in `tests/acceptance/_output` - a `.gitignore` file in the `_groups` directory containing:
```
*
!.gitignore
```

Reply:
Done